proxy-node-translator/src/main/java/uk/gov/ida/notification/translator/resources/HubResponseTranslatorResource.java

```diff
@@ -57,7 +58,14 @@ public Response hubResponse(@Valid HubResponseTranslatorRequest hubResponseTrans
                         )
                 );
 
-        logResponseAttributesHash(hubResponseTranslatorRequest, translatedHubResponse);
+
+        EidasAttributesLogger eidasAttributesLogger = new EidasAttributesLogger(EidasResponseAttributesHashLogger::instance, null);
```

Comment:
I'm not sure this is the best way to instantiate the logger - as in the caller shouldn't know or care that there are different methods of instantiating the logger depending on which method of hashing is desired (e.g. that the second arg has to be null in this case). This could be abstracted away further. But this is okay for now and we can refactor this later.